test/library/standard/Set/leaks/doubleStringAdd.future

```diff
@@ -0,0 +1 @@
+bug: this test leaks memory
```

Comment:
This should have a backing issue to track it, and a mention of that issue here.  I also might recommend some instructions for what to do when the test passes (e.g. "remove the .execopts so it runs everywhere" or something along those lines)

Reply:
Since it sounds like you'll be working on this, this isn't as necessary, but it'll still be helpful if you end up getting stuck and needing to come back to it later or something